components/monitoring/grafana/README.md

```diff
@@ -18,50 +18,23 @@ First, create the `appstudio-grafana` namespace on each Prometheus or Grafana cl
 $ oc create namespace appstudio-grafana
 ```
 
-### Grafana Datasources
+### Grafana Datasource
 
-Grafana datasources contain the connection settings to the Prometheus instances. These datasources are stored in secrets in the `appstudio-workload-monitoring` namespace of the **Grafana cluster**.
+Grafana datasources contain the connection settings to the Prometheus instances.
 
-The Prometheus endpoints called by Grafana are protected by an OAuth proxy running as a sidecar container and which checks that the incoming requests contain a valid token. A token is valid if it belongs to a service account of the **Prometheus cluster** which has the RBAC permission to "get namespaces". Such a permission can be obtained with the `cluster-monitoring-view` cluster role.
+A single [kustomization file](https://github.com/redhat-appstudio/infra-deployments/blob/main/components/monitoring/grafana/base/kustomization.yaml#L14)
```

Comment:
I'm not sure I'm able to follow this paragraph. How is that kustomization file lets us query those metrics and how is that related to data sources.

Reply:
Moved to Monitoring README